src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+function removeBook(state, id) {
+  if (id === state.length - 1) return state.slice(0, id);
+
+  if (id === 0) return state.slice(1);
+
+  return [...state.slice(0, id), ...state.slice(id + 1)];
+}
```

Comment:
As part of the project requirements, kindly make sure your removeBook function uses the ES6 filter() method to create a new array, which will not contain the book you want to remove from the store (for instance you can filter by the id key - i.e.: return state.filter(book => book.id !== id);